tests/template/srw_resize_3d_test.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+"""PyTest for :mod:`sirepo.template.srw`
+
+:copyright: Copyright (c) 2023 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from pykern.pkcollections import PKDict
```

Comment:
Remove these imports and add an import pytest

Reply:
I see `import pytest` all over but it is rarely used. Also I still have to do the original imports in the test function. What does pytest do?

Reply:
The import pytest brings in pytest (the testing framework) as the first thing in the program. It may be superstition at this point, but it used to do goofy things with "fixtures" when you didn't import pytest. Perhaps we can dispense with it. Let's try for this test.